model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -170,32 +173,37 @@ def sample(
         HospAdmissionsSample
         """
 
-        infection_hosp_rate, *_ = self.infect_hosp_rate_dist.sample(**kwargs)
+        infection_hosp_rate, *_ = self.infect_hosp_rate_rv.sample(**kwargs)
 
-        infection_hosp_rate_t = infection_hosp_rate * latent
+        infection_hosp_rate_t = infection_hosp_rate * latent_infections
 
         (
-            infection_to_admission_interval,
+            infection_to_admission_interval_rv,
             *_,
-        ) = self.infection_to_admission_interval.sample(**kwargs)
+        ) = self.infection_to_admission_interval_rv.sample(**kwargs)
 
-        predicted_admissions = jnp.convolve(
-            infection_hosp_rate_t, infection_to_admission_interval, mode="full"
+        observed_hosp_admissions = jnp.convolve(
+            infection_hosp_rate_t,
+            infection_to_admission_interval_rv,
+            mode="full",
         )[: infection_hosp_rate_t.shape[0]]
 
         # Applying weekday effect
-        predicted_admissions = (
-            predicted_admissions * self.weekday_effect_dist.sample(**kwargs)[0]
+        observed_hosp_admissions = (
+            observed_hosp_admissions
+            * self.weekday_effect_rv.sample(**kwargs)[0]
         )
 
         # Applying probability of hospitalization effect
-        predicted_admissions = (
-            predicted_admissions
-            * self.hosp_report_prob_dist.sample(**kwargs)[0]
+        observed_hosp_admissions = (
+            observed_hosp_admissions
+            * self.hosp_report_prob_rv.sample(**kwargs)[0]
```

Comment:
Low confidence in this remark, but my intuition is that one of the two `observed_hosp_admissions` can be changed for clarity. Perhaps the `observed_hosp_admissions` in `()` can be changed to `hosp_admissions` &rarr;

```python
observed_hosp_admissions = (
    hosp_admissions
    * self.hosp_report_prob_rv.sample(**kwargs)[0]
```

Reply:
yes, I think one of these should be `latent_hosp_admissions`